api/v1/services/CheckInService.js

```diff
@@ -22,7 +23,16 @@ module.exports.findCheckInByUserId = function (userId){
                 var source = "userId";
                 throw new errors.NotFoundError(message, source);
             }
-            return checkin;
+            return NetworkCredential.findByUserId(userId)
+            .then(function(credentials){
+                if (!_.isNull(credentials)) {
```

Comment:
If it's `null`, why not just set `credentials` in the object to `credentials` anyways?